### ISSUE TITLE
Should be able to kubicorn list

### ISSUE THREAD

**Issue by geojaz** [kris-nova/kubicorn#9]
Should return the list of available states.

**Comment by k4k** [comment]
The clarify, are you looking for a list of the states in _state? Example:
```
$ kubicorn list
adorable-sheepdog
amazing-painting
attractive-creature
attractive-sculpture
```

**Comment by kris-nova** [comment]
@k4k how is this PR coming, anything I can help with?

**Comment by geojaz** [comment]
Closed in #33!
